src/ccm/ng-ce-ui/values.yaml

```diff
@@ -0,0 +1,78 @@
+# Default values for ng-ce-ui
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+global:
+  loadbalancerURL: "https://test"
+  ingress:
+    enabled: false
+    className: harness
+    hosts:
+      - my-host.example.org
+    tls:
+      enabled: true
+      secretName: ''
+  istio:
+    enabled: false
+    gateway:
+      create: false
+    virtualService:
+      gateways:
+      hosts:
+
+replicaCount: 2
+
+maxSurge: 100%
+
+maxUnavailable: 0
+
+image:
+  registry: docker.io
+  repository: harness/ng-ce-ui
+  pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "0.22.3"
+  digest: ""
+
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  create: false
+  annotations: {}
+  name: "harness-default"
+
+podAnnotations: {
+  cluster-autoscaler.kubernetes.io/safe-to-evict: "true"
```

Comment:
Is this required?

Reply:
Not sure why I added it, maybe I was little sleepy. 😄 Removed it.